source/speech/__init__.py

```diff
@@ -790,7 +826,25 @@ def _speakTextInfo_addMath(speechSequence, info, field):
 	except (NotImplementedError, LookupError):
 		return
 
-def speakTextInfo(info, useCache=True, formatConfig=None, unit=None, reason=controlTypes.REASON_QUERY, _prefixSpeechCommand=None, onlyInitialFields=False, suppressBlanks=False,priority=None):
+
+def _logBadSequenceTypes(sequence, shouldRaise: bool = True) -> bool:

```

Comment:
Why is this wrapper here? It seems a bit overkill, why not use speech.types.logBadSequenceTypes directly?

Reply:
It just made it easier to change how logging was handled in all of speech. Add extra debugging etc. I'll remove it now.